test/cri-containerd/createcontainer_test.go

```diff
@@ -72,6 +73,39 @@ func runCreateContainerTestWithSandbox(t *testing.T, sandboxRequest *runtime.Run
 	}
 }
 
+func execContainerTestWithSandbox(t *testing.T, sandboxRequest *runtime.RunPodSandboxRequest, request *runtime.CreateContainerRequest, execReq *runtime.ExecSyncRequest) *runtime.ExecSyncResponse {
```

Comment:
Should we make this just exec within a sandbox? Seems like its easier for the tests to have flexibility then. Does that make sense?

Reply:
Yeah. makes sense. I will update this

Reply:
Awesome thanks!